src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs

```diff
@@ -1520,6 +2042,26 @@ private static Vector128<byte> ExtractAsciiVector(Vector128<ushort> vectorFirst,
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static Vector256<byte> ExtractAsciiVector(Vector256<ushort> vectorFirst, Vector256<ushort> vectorSecond)
+        {
+            // Narrows two vectors of words [ w7 w6 w5 w4 w3 w2 w1 w0 ] and [ w7' w6' w5' w4' w3' w2' w1' w0' ]
+            // to a vector of bytes [ b7 ... b0 b7' ... b0'].
+
+            // prefer architecture specific intrinsic as they don't perform additional AND like Vector512.Narrow does
```

Comment:
I think this comment is out of sync. We're using the non architecture specific Narrow below.

Likewise, given we're just calling the xplat narrow, do we need this helper method?